add State Maintenance, mostly handwaving text. (#16)
danwing/cidfi: draft-wing-cidfi.md

belongs in the QUIC transport parameter signaling?

# State Maintenance

> The CNE can safely remove state after UDP inactivity timeout {{Section

Reviewer: ```suggestion
A CNE can safely remove state after UDP inactivity timeout ({{Section
```

> 4.3 of !RFC4787}}.  The CIDFI client MUST re-signal its CNE when it

Reviewer: ```suggestion
4.3 of !RFC4787}}).  The CIDFI client MUST re-signal its CNE(s) when it
```

receives a QUIC path validation message, as that indicates a NAT

> rebinding occurred.  CNE state can also be cleared by signaling from

Reviewer: ```suggestion
rebinding occurred.  A CNE's state can also be cleared by signaling from
```
